lib/ddtrace/contrib/rails/configuration/settings.rb

```diff
@@ -58,32 +58,32 @@ def initialize(options = {})
             o.default 'views/'
             o.on_set do |value|
               # Update ActionView template base path too
-              Datadog.configuration[:action_view][:template_base_path] = value
+              Datadog::Tracing.configuration[:action_view][:template_base_path] = value
             end
           end
 
           DEPRECATION_WARN_ONLY_ONCE_TRUE = Datadog::Utils::OnlyOnce.new
           DEPRECATION_WARN_ONLY_ONCE_FALSE = Datadog::Utils::OnlyOnce.new
 
           option :log_injection do |o|
-            o.delegate_to { Datadog.configuration.log_injection }
+            o.delegate_to { Datadog::Tracing.configuration.log_injection }
             o.lazy
             o.on_set do |value|
               if value
                 DEPRECATION_WARN_ONLY_ONCE_TRUE.run do
                   Datadog.logger.warn(
                     "log_injection is now a global option that defaults to `true`\n" \
                     "and can't be configured on per-integration basis.\n" \
-                    'Please remove the `log_injection` setting from `c.use :rails, log_injection: ...`.'
+                    'Please remove the `log_injection` setting from `c.instrument :rails, log_injection: ...`.'
                   )
                 end
               else
                 DEPRECATION_WARN_ONLY_ONCE_FALSE.run do
                   Datadog.logger.warn(
                     "log_injection is now a global option that defaults to `true`\n" \
                      "and can't be configured on per-integration basis.\n" \
-                    'Please remove the `log_injection` setting from `c.use :rails, log_injection: ...` and use ' \
-                    "`Datadog.configure { |c| c.log_injection = false }` if you wish to disable it.\n"
+                    'Please remove the `log_injection` setting from `c.instrument :rails, log_injection: ...` and use ' \
+                    "`Datadog::Tracing.configure { |c| c.log_injection = false }` if you wish to disable it.\n"
                   )
                 end
               end
```

Comment:
Perhaps just delete it instead? Like another similar above, this is already deprecated anyway.